packages/dom/src/dom/is-edge.js

```diff
@@ -7,16 +7,7 @@ import getRectangleFromRange from './get-rectangle-from-range';
 import isSelectionForward from './is-selection-forward';
 import hiddenCaretRangeFromPoint from './hidden-caret-range-from-point';
 import { assertIsDefined } from '../utils/assert-is-defined';
-
-/* eslint-disable jsdoc/valid-types */
-/**
- * @param {Element} element
- * @return {element is HTMLInputElement | HTMLTextAreaElement} Whether the element is an input or textarea
- */
-function isInputOrTextArea( element ) {
-	/* eslint-enable jsdoc/valid-types */
-	return element.tagName === 'INPUT' || element.tagName === 'TEXTAREA';
-}
+import isInputOrTextArea from './is-input-or-text-area';
```

Comment:
Minor naming nit: I realize that it's called `isInputOrTextArea` and not `isInputOrTextarea`, but to me `textarea` is a one-word term, so I expect that to be reflected at least in the file name (I realize we can't change `HTMLTextAreaElement`). What do you think?

Reply:
I wrote it to align with the type, `HTMLTextAreaElement` as you mentioned, but also initially had it as a single word... however I wanted to make it easier to remember which casing was being used so I decided on the one we couldn't change 🙂 Happy to change it if you feel strongly about it though.

Reply:
I don't feel so strongly about it, but it just keeps bugging me, lol. If we leave it in 2 words, it's inconsistent with the `tagName`. If we change it to 1 word, it'd be inconsistent with the type name. Guess there's no good way around it 😅 

That being said, feel free to leave it as-is 👍

Reply:
Don't worry, it bugs me too. I'm glad to be in good company 😂